src/buildGrid.js

```diff
@@ -0,0 +1,22 @@
+function buildGrid(articles) {
+  var root = document.getElementById("root");
+  var thumbs = document.createElement("div");
+  root.innerHTML = "<h1>Ame News</h1>";
+
+  root.appendChild(thumbs);
+  thumbs.setAttribute("class","thumbs");
+  thumbs.setAttribute("id","thumbs");
+
+  articles.forEach(function(item, index) {
```

Comment:
Feels like this could be abstracted into a buildArticle method and then you could just do:
```
articles.forEach(buildArticle);
```
And you would have documented the intent more clearly.